namespaces/live-1.cloud-platform.service.justice.gov.uk/hello-world-test/00-namespace.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: hello-world-test
+  labels:
+    cloud-platform.justice.gov.uk/is-production: "false"
+    cloud-platform.justice.gov.uk/environment-name: "development"
+  annotations:
+    cloud-platform.justice.gov.uk/business-unit: "HMPPS Digital"
+    cloud-platform.justice.gov.uk/slack-channel: "${slack-channel}"
```

Comment:
Please replace `${slack-channel}` with the name of a team slack channel we can use to get in touch if there are any problems with this namespace, e.g. `#hmpps_dev`

Reply:
Hi, I have updated and added a private channel. Please just let me know if the team channel needs to be public?